src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms ease-in;
```

Comment:
Don't know if it makes a difference for the task, but ease-in can be left out, also `transition: all` should be used carefully

Reply:
Yes, use individual properties instead of `all`.
`all` should be used if you transition 4 or more rules at the same time.